daemons/gptp/common/avbts_port.hpp

```diff
@@ -202,11 +207,95 @@ class PortIdentity {
 typedef std::map < PortIdentity, LinkLayerAddress > IdentityMap_t;
 
 /**
+ * Structure for initializing the IEEE1588 class
+ */
+typedef struct {
+	/* clock IEEE1588Clock instance */
```

Comment:
It turns out that this is an *incredibly* common thing. I did a regex search for Doxygen-style comments without any Doxygen tags and fixed them all, but it ended up touching most of the .h/.hpp files in the gptp daemon source. It may be best to make a separate pull request for it afterward.